JavaScript with Axios/get-resource-id-endpoint.js

```diff
@@ -0,0 +1,18 @@
+// This request demonstrates how to retrieve a resource using an ID.
+const axios = require('axios');
+
+let config = {
+  method: 'get',
+  maxBodyLength: Infinity, // set maximum length of the request body
+  url: 'https://api.pdfrest.com/resource/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx?format=url', // Replace with your API key
```

Comment:
This looks incorrect -- the resource ID ought to go here, not the API key?

Reply:
Also, is there a way to return something more useful than the exact same URL?

Reply:
> Also, is there a way to return something more useful than the exact same URL?

Are you asking from a front-end or a back-end perspective?

> This looks incorrect -- the resource ID ought to go here, not the API key?

Unless we've changed from UUIDs they are a standard length, this is also the same format as our [production docs](https://pdfrest.com/documentation.html#/resource/{id})

Reply:
> Unless we've changed from UUIDs they are a standard length, this is also the same format as our [production docs](https://pdfrest.com/documentation.html#/resource/%7Bid%7D)

The documentation indicates that the resource ID is required, not the API key. The comment on line 7 should be changed.

> Are you asking from a front-end or a back-end perspective?

I was asking from a front-end perspective. But now I see what it's doing: it returns a URL to get the file format. I missed that initially.

Reply:
Oh! You're referring to the comment, thanks!